### ISSUE TITLE
hi,when i run test,a error :unknown loss function:loss_func.What is going on, can it be solved? 

### ISSUE THREAD

**Issue by haixinxiaopengyou** [JusperLee/Looking-to-Listen-at-the-Cocktail-Party#21]

**Comment by azuredsky** [comment]
> _No description provided._

Hi，haixinxiaopengyou  
u can check this code:
```
from model.model.loss import audio_discriminate_loss, audio_discriminate_loss2
av_model = load_model(model_path,custom_objects={'tf':tf, 'loss_func':audio_discriminate_loss})
```

it is worked for me

**Comment by SutirthaChakraborty** [comment]
Downgrade the h5py
`!pip install 'h5py==2.10.0' --force-reinstall
`
`AV_model = load_model(model_path,custom_objects={"tf": tf},compile=False)
adam = optimizers.Adam()
AV_model.compile(optimizer=adam, loss='mse')`
